content/blog/ways-to-interact-with-kubernetes-clusters-managed-by-hpe-ezmeral-container-platform.md

```diff
@@ -0,0 +1,180 @@
+
+
+
+
+## WebUI
+The first method by which you can interact with a Kubernetes Clusters managed by HPE ECP is, of course, through the web user interface (WebUI).
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Cluster.png)     | In the Main Menu, if you are the administrator of the HPE Ezmeral CP, you can manage and monitor the status of the K8s clusters managed by HPE ECP.  |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Dashboard.png)   | You can access the Kubernetes Dashboard just as you would do with open source Kubernetes.  |
+
+## WebTerminal
+Inside the Kubernetes Tenant, at the bottom, there is a web terminal for you to interact with Kubernetes using the ```kubectl``` command.
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant.png)      | At the bottom, you can click "Initialize" to initiate the web terminal instance.  |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant-02.png)   | Type your ``` kubectl ``` command to interact with Kubernetes.  |
+
+
+## HPE Kubectl Plugin
+If you want to use kubectl remotely, you must first install the kubectl-plugin to get the session ID to access HPE Ezmeral. To install the kubectl-hpecp plugin, you can run the following command. The HPE kubectl plugin is required to establish the tenant’s authenticated kubectl requests. Therefore, if you want to use kubectl remotely as a tenant user, you must first install the HPE kubectl plugin (kubectl-hpecp) to get an HPE ECP authentication token (a session ID), fetch the kubconfig manifest file, and then interact with the managed K8s clusters.
+
+To install the kubectl-hpecp plugin, first fetch the plugin from the WebUI. Log in as a tenant user, and click on Download HPE Kubectl Plugin (as shown in the picture below) and download the plugin and installation instructions according to your target operating system. The following codes are executed in Linux environment.
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant-03.png)      | You can download the required binary here as well.  |
+
+### Installation of ```kubectl``` and ```kubectl-hpecp```
+#### Step 1: Make sure you have ```kubectl``` install.
+```bash
+# Download the latest version of kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+# And place it anywhere in your PATH:
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+> Reference:

```

Comment:
Even though there is only one reference, the references section should all be formatted consistently. In this case, you would have one bullet below "Reference:"